dvc/repo/metrics/show.py

```diff
@@ -151,7 +151,7 @@ def _read_metrics(self, metrics, branch):
     return res
 
 
-def show(
+def get_metrics(
```

Comment:
Let's keep it as `show` for symmetry with CLI.

Reply:
Also, it is part of the API that we are trying to keep stable for future official release :slightly_smiling_face:

Reply:
Hmm, so maybe we should consider moving display functionality inside this method?  Because now it looks like it might misguide user. If that will stay as part of official API, that can cause some issues, like "dvc show does not show"

Reply:
Yes, we will definitely consider moving it into API as a `repo.metrics.show(display=True)`(or smth like that) in the future. I just don't feel like it is the scope of this PR.

Reply:
Ok, Ill reverse naming change.